packages/validator/test/cli-validator/tests/config-file-validator.test.js

```diff
@@ -115,14 +115,17 @@ describe('cli tool - test config file validator', function() {
     const res = configFileValidator(config, chalk);
     const capturedText = getCapturedText(consoleSpy.mock.calls);
 
+    // let text = '';
+    // for (i = 0; i < capturedText.length; i++) {
+    //   text += `[${i.toString()}]: ${capturedText[i]}\n`;
+    // }
+    // console.warn(text);
+
```

Comment:
What about these commented-out lines?

Reply:
I had added this code while debugging test failures, but then commented it out after I got the tests working again.  I'll remove.